### ISSUE TITLE
Improving tab navigation and adding alt texts

### ISSUE THREAD

**Issue by kclalit64** [linkcollect/Main-Website-Linkcollect#177]
The applications as of now, lacks some essential accessbility principles as per standard web dev practices like alt texts and tab navigation:

1. Tab navigation: Currently, it's just the default tab navigation that comes shipped with the standard HTML DOM and it's not making much sense for now. I'll handle this.
![Screenshot from 2023-10-08 00-53-16](https://github.com/linkcollect/Main-Website-Linkcollect/assets/91486778/c35b14c8-9137-4f67-8ca8-789823d535c7)
2. ALT Text: Describing certain elements via add on texts to help visually impaired people read faster on their screen readers.

Please assign me this issue if you feel like going forward with it. Thanks!

**Comment by Vishusingh16** [comment]
I will be thankful if you assign me with this issue

**Comment by kclalit64** [comment]
bruh i literally mentioned in the description to assign me the issue lol. kya karna hai issues churake; khud se enhancements, potential features ya bugs dhund aur solve kar zyada sikhne ko milega

**Comment by Harsh8055** [comment]
hey @kclalit64 , " it's just the default tab navigation that comes shipped with the standard HTML DOM and it's not making much sense for now. I'll handle this.

"Do you mean swithcing between Home explore ands saved ? sure you can start on this once we understand what exactly is needed

2 - sure, go ahead i will assgin it to you

**Comment by kclalit64** [comment]
Yeah the default tab navigation often isn't the most optimal as in this case. It's like to help access elements faster than a regular drag and drop in some order that would generally make sense. For further clarification, you could refer https://www.csun.edu/universal-design-center/web-accessibility-criteria-tab-order. 

lmk if that feels important to you. Anyways, will proceed with 2.

**Comment by Harsh8055** [comment]
i see, once 2 is done we can start addding tab orders as well :)
